wurst/dummy/DummyRecycler.wurst

```diff
@@ -204,9 +204,12 @@ init
 	..setAttack1DamageSidesperDie(1)
 	..setAttack1ProjectileSpeed(INT_MAX)
 	..setAttack1Range(INT_MAX)
-	..setAttack1TargetsAllowed(commaList(TargetsAllowed.air, TargetsAllowed.debris, TargetsAllowed.ground, TargetsAllowed.invulnerable, TargetsAllowed.item_t, TargetsAllowed.structure, TargetsAllowed.vulnerable, TargetsAllowed.ward))
+	..setAttack1TargetsAllowed(
+		commaList(TargetsAllowed.air, TargetsAllowed.debris, TargetsAllowed.ground, TargetsAllowed.invulnerable,
```

Comment:
Drive-by fix: minor tidying to use 120-char line limit

Happy to revert if needed